tests/config/dapr_mqtt_pubsub.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: mqtt-pubsub
+spec:
+  type: pubsub.mqtt
+  version: v1
+  metadata:
+  - name: url
+    value: "tcp://mosquitto.dapr-tests.svc.cluster.local:1883"
```

Comment:
Why bringing MQTT to the test?

Reply:
while redis does support wildcards when using `psubscribe`, dapr uses `XGroupCreateMkStream` (https://github.com/dapr/components-contrib/blob/6e4139c7b1da21ea4a5d14d2e323a7ff37ee2d9f/pubsub/redis/redis.go#L172), which does not support wildcards.